### PR TITLE
Add the Marshmallow joke in the Random directory.

### PR DESCRIPTION
**Pull request by lamprof**
I created a new file for the Marshmallow joke.

### CODE REVIEW

**Comment by vramdhanie** [comment]
This is a classic joke. I really enjoyed it.

**Comment by martelr** [comment]
awesome!

**Comment by lamprof** [comment]
Thanks !

File: Random/marshmallow.md [hunk 1]
```diff
@@ -0,0 +1 @@
+I dreamt I was forced to eat a giant marshmallow. When I woke up, my pillow was gone.
```
Comment (vramdhanie) [inline Random/marshmallow.md:1]: Is this line of code too long?

Comment (frenetj) [reply Random/marshmallow.md:1]: Looks fine to me...

**Pull request merged**